openstudiocore/CMake/FindEnergyPlus.cmake

```diff
@@ -50,9 +50,9 @@ else()
   file(GLOB ENERGYPLUS_POSSIBLE_PATHS "/usr/local/[eE]nergy[pP]lus?*")
 endif()
 
-# sort possible paths in reverse order
-list(APPEND ENERGYPLUS_POSSIBLE_PATHS $ENV{ENERGYPLUSDIR})
+# sort possible paths in reverse order, favoring the ENERGYPLUSDIR environment variable
```

Comment:
Is that something people do?  OpenStudio is currently locked to the EnergyPlus SHA, so someone would have to change that too.